dlt/extract/hints.py

```diff
@@ -208,6 +209,14 @@ def apply_hints(
                 t["incremental"] = None
             else:
                 t["incremental"] = incremental
+        if additional_table_hints is not None:
+            # loop through provided hints and add, overwrite, or remove them
+            for k, v in additional_table_hints.items():
+                if v:
```

Comment:
This has to be "if v is not None" otherwise a value of false or [] will also be removed although it needs to stay in.

Reply:
I adjusted it.

I had it like that to maintain consistency with how the standard table hints are handles (remove when "empty value" is provided), but I agree that isn't desired behavior for the additional table hints.